<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>25</number>
    </property>
    <property name="leftMargin">
     <number>30</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>30</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QLabel" name="label_title">
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Регистрация&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;нового пользователя&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_logging">
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(73, 168, 0);

border-color: rgb(255, 255, 255);
border-radius: 10px;
border: 1px solid white;

padding: 10px;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Введите логин</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(73, 168, 0);

border-color: rgb(255, 255, 255);
border-radius: 10px;
border: 1px solid white;

padding: 10px;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Введите пароль</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_reset">
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: rgb(255, 255, 255);

padding: 5px;

border-color: rgb(255, 255, 255);
border-radius: 10px;
border: 1px solid white;


}

QPushButton:hover {
background-color: rgb(4, 74, 0);
}</string>
      </property>
      <property name="text">
       <string>Сброс</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <property name="spacing">
       <number>50</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_back">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
color: rgb(255, 255, 255);

padding: 10px;

border-color: rgb(255, 255, 255);
border-radius: 15px;
border: 1px solid white;

}

QPushButton:hover {
background-color: rgb(4, 74, 0);
}</string>
        </property>
        <property name="text">
         <string>Назад</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_registration">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
color: rgb(255, 255, 255);

padding: 10px;

border-color: rgb(255, 255, 255);
border-radius: 15px;
border: 1px solid white;

}

QPushButton:hover {
background-color: rgb(4, 74, 0);
}</string>
        </property>
        <property name="text">
         <string>Регистрация</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
